frontend/src/pages/Login.tsx

```diff
@@ -1,7 +1,40 @@
+import React from 'react';
+import GoogleLogin from 'react-google-login';
 import './Login.css';
+import axios from 'axios';
 
 function Login() {
-    return <h1>Login</h1>;
+    const onFailure = () => {
+        console.log('Invalid account');
+    };
+
+    const onSuccess = (googleData) => {
+        axios
+            .post('http://localhost:5000/api/users/login', {
+                credential: googleData.tokenId,
+            })
+            .then(function (response) {
+                console.log(response.data);
+            });
```

Comment:
I feel like using an arrow function here would be nicer (Even though this is just for testing)